When you build, you'll get errors like:

```
(gsoc)andrew@kaim-eeyore:~/git/gsoc/PRU-I2C_SPI_master$ make CGTDIR=~/ti-cgt-pru_2.1.2
clpru -v3 -s -al -O3 --c99 --gcc --printf_support=minimal --symdebug:none -I/home/andrew/ti-cgt-pru_2.1.2/include -I/home/andrew/ti-cgt-pru_2.1.2/lib -I./include -ppd -ppa -c pru0_spi.c -ea=.s
"pru0_spi.c", line 26: warning: function "scanf" declared implicitly
"pru0_spi.c", line 106: warning: last line of file ends without a newline
clpru -v3 -s -al -O3 --c99 --gcc --printf_support=minimal --symdebug:none -I/home/andrew/ti-cgt-pru_2.1.2/include -I/home/andrew/ti-cgt-pru_2.1.2/lib -I./include -ppd -ppa pru0_spi.obj -q -z -cr --diag_warning=225 -lam335x_pru.cmd -x -i/home/andrew/ti-cgt-pru_2.1.2/lib -o pru0_spi
"am335x_pru.cmd", line 72: error: program will not fit into available memory.
   placement with alignment fails for section ".text" size 0x239c page 0.
   Available memory ranges:
   PRU_IMEM     size: 0x2000       unused: 0x1fe4       max hole: 0x1fe4    
error: errors encountered during linking; "pru0_spi" not built

>> Compilation failure
make: *** [pru0_spi] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -29,7 +29,7 @@ CFLAGS= -v3 -s -al -O3 --c99 --gcc --printf_support=minimal --symdebug:none $(IN
 LDFLAGS=-cr --diag_warning=225 -lam335x_pru.cmd -x -i$(CGTDIR)/lib
 
 pru0_spi: pru0_spi.obj
-	$(CC) $(CFLAGS) $^ -q -z $(LDFLAGS) -o $@
+	$(LD) $(LDFLAGS) $^ -o $@
 
 %.obj: %.c
 	$(CC) $(CFLAGS) -c $< -ea=.s

